tools/const_finder/test/fixtures/lib/target.dart

```diff
@@ -0,0 +1,14 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+class Target {
+  const Target(this.stringValue, this.intValue);
+
+  final String stringValue;
+  final int intValue;
+
+  void hit() {
+    // call to prevent tree shaking.
```

Comment:
you could perform some side-effect here like printing to more easily guarantee a future (I hope...) compiler optimization doesn't determine that this is still a no-op and tree shake it

Reply:
Doing anything from dart:core was throwing errors during kernel compilation - was I missing some argument to make that work?

Originally I was trying to use print.

Reply:
Fixed up the compilation so I can use print.